### DISCUSSION TITLE
TileMap collision detection requirements are a bit strange

### DISCUSSION THREAD

**Discussion by SanctusAnimus** [godotengine/godot-proposals#8279] [2D]
So i've been learning how to use TileMaps, with physics layer etc and made an interesting discovery: my Area2D's cannot detect collisions with TileMap unless they are monitorable.

And it would make sense to me if it also required proper setup of layers and masks - but no! My projectiles (extending Area2D) have no layer set, they are intended to only scan other layers:
![image](https://github.com/godotengine/godot-proposals/assets/43399078/018c2bff-0916-4dfa-91b0-67cfb3ea591f)
While TileMap has physics layer with no mask:
![image](https://github.com/godotengine/godot-proposals/assets/43399078/a4459c98-8123-4a99-ae87-6d022874ff7e)

And with `Monitorable` set to false on Area2D, collision signals are never emitted on it, as if only TileMap is processing collisions, ignoring masks and layers. Why? 

Asking here instead of creating and issue cause i have no idea whether this is intended.

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot/issues/17238,
https://github.com/godotengine/godot/issues/66468,
and https://github.com/godotengine/godot/issues/71489 (possible duplicate).
